### PR TITLE
Milestone six

### PR DESCRIPTION
**Pull request by mesielepush**

### CODE REVIEW

**Comment by mesielepush** [comment]
Two columns for friendship, to upgrade friends search.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Change required :red_circle: 

Hello @mesielepush 

THe project is almost OK. 
There is that error after accepting the friendship request.
Screenshot from 2020-02-13 01-28-14

Kindly, fix that and submit a new code review request.
Thanks

Mael FOSSO

**Comment by samgaco** [comment]
Hi @mesielepush :high_brightness: 

I have tried to verify the previous requirement, but I seem not to be able to send a friend requests:


request

Please submit back once that is fixed. 

- [x] Every user on the list should[ show an option at the side to send friend requests](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/userctr.png), and a user should not be able to see itself in such list.


--------------------

* ###### STATUS OF THE MILESTONE: REQUESTED CHANGES :stop_sign:

**Comment by mesielepush** [comment]
That is not the UI of mine, I don't know where that caption of yours came from. The friendship request is on the profile of every user. But I'll will put it on the friends index as well.

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+      <% if @user.gravatar_url != nil %>
+        <%= image_tag( @user.gravatar_url, width: 120, height: 120 ) %>
+      <% end  %>
+
+  <% end %>
+  
   <article class="timeline">
-    <h3>Recent posts:</h3>
+    
+    <h3> Lastest %> </h3>
```
Comment (maelfosso) [inline app/views/users/show.html.erb:47]: Correct that text

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,9 +1,54 @@
...
+      <% else  %>
+        <% if current_user.friend_requests.include?(@user.id) %>
+          <%= link_to 'confirm friendship', confirm_friend_url( user_id: @user.id, friend_id: current_user.id), class:'btn btn-info' %>
+        
+        <% elsif current_user.pending_friends.include?(@user.id) %>
+          <p>
+            wainting for response
+          <p>
+        <% else  %>
+          <%= link_to 'request_friendship', request_friendship_path(user_id: current_user.id, friend_id: @user.id), class:'btn btn-info' %>
```
Comment (maelfosso) [inline app/views/users/show.html.erb:34]: Don't you want to write `request friendship` here?

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

Well done job :+1: 
You can merge it :smile: 

Good luck with the next milestone

Mael FOSSO

**Pull request merged**